docker-compose.yml

```diff
@@ -53,6 +55,13 @@ services:
       - lnd-outside-3
       - otel-agent
     restart: on-failure:10
+  apollo-router:
+    image: ghcr.io/apollographql/router:v1.25.0
+    environment:
+      - APOLLO_ROUTER_SUPERGRAPH_PATH=/repo/dev/apollo-federation/supergraph.graphql
```

Comment:
I think we don't typically hard code stuff in here? instead we use env variable?

Reply:
we do actually hard code paths to config.
See Otel agent for example:
```
  otel-agent:
    image: otel/opentelemetry-collector-contrib:0.79.0
    command: ["--config=/etc/otel-agent-config.yaml"]
    environment:
      - HONEYCOMB_DATASET=${HONEYCOMB_DATASET}
      - HONEYCOMB_API_KEY=${HONEYCOMB_API_KEY}
    volumes:
      - ./dev/otel-agent-config.yaml:/etc/otel-agent-config.yaml
```